### ISSUE TITLE
Some things seem to be gone with the new sheets

### ISSUE THREAD

**Issue by Liviu50** [foundryvtt/dnd5e#3218]
So, i love all of your efforts, especially to add dnd 5e support to foundry cause i love using it there.
So i super appreciate your effort and cheer you on.
That being said the new one has some stuff missing, or maybe is there a setting i am missing?
items do not have anymore all the components that items had.
for example you can no longer check, find or know if a weapon is 2 handed, light or versatile.
Spells same thing...which spell requires conc? which does require spoken words or gestures?
Are those still WIP? cause i would seem to me like something essential to have since....well they are basic things all items and spells should have cause its somewhat important.
As for tools...why is the space for "special" gone? i mean if the weapon prof has it...and the armor has it...why not add that to all the stuff like resistances, immunities, and also tools so you can put your own in case there are some extras? 
I also noticed that some classes seem to break easily...like the monk gets 0 movement because they are given the unarmored movement but also stunning strike i guess? which makes speed go to 0 and every movement becomes impossible to modify.
I was mostly curious cause the new version seems somewhat downgrade from the old one in terms of usability.
it looks awesome, we love it and testing it in an active campaign and we would love to se it evolve but it seem to be missing basic stuff it had in the previous version...
i mean sure let us ignore the bugs and monk issues....those will be fixed.
but the spell and items things are a tad perplexing...any plans on that matter?
i would love to know more about the situation this awesome project is on ^^
Cheers and have a nice day everyone!

**Comment by krbz999** [comment]
> for example you can no longer check, find or know if a weapon is 2 handed, light or versatile.

There has not been any change to item sheets. These should all still be available to pick right there in the 'Details' tab.

Pretty hard to follow what else you are describing, but I'll answer the question about spell components with the below image:

![image](https://github.com/foundryvtt/dnd5e/assets/50169243/a3f01a76-905b-40e5-960c-340ebe7a7e10)

I think the rest may be describing behaviors from several, possibly defunct, modules.

**Comment by Liviu50** [comment]
maybe its because of the compendium spells? or items? cause truly i must create a new world from scratch to see those....evne if they use same system.
huh thx ill look into it.
